api-samples/sandbox/sandbox/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Sandboxed Frame Sample",
+  "version": "1.0.3",
+  "manifest_version": 3,
+  "minimum_chrome_version": "23",
+  "background": {
+      "service_worker": "main.js"
```

Comment:
Hey @IanStanion-google, do you think it would be a good idea to use service-worker.js for all the samples for consistency purposes?

Reply:
Yes, absolutely. This was an older sample I fixed up, sorry to miss that!